### PR TITLE
Copy changed yaml file to github

### PR DESCRIPTION
**Pull request by borg323**
An untested suggestion to copy changed yaml file to a github branch every time training starts. If the yaml file is unchanged the git operation should not do anything.

### CODE REVIEW

File: tf/start.sh [hunk 1]
```diff
@@ -42,6 +42,10 @@ do
     # train all networks
     for netarch in ${NETARCHS[@]}
     do
+      echo "Saving .yaml changes:"
+      git add $CONFIGDIR/$netarch.yaml
+      git commit -m "Configuration change"
+      git push https://github.com/LeelaChessZero/lczero-training.git running_conf
```
Comment (Error323) [inline tf/start.sh]: I think this is good indeed!
Probably the last command should be performed in parallel using `&`.

Comment (borg323) [reply tf/start.sh]: Done

File: tf/start.sh [hunk 2]
```diff
@@ -42,6 +42,10 @@ do
     # train all networks
     for netarch in ${NETARCHS[@]}
     do
+      echo "Saving .yaml changes:"
+      git add $CONFIGDIR/$netarch.yaml
```
Comment (Tilps) [inline tf/start.sh]: This only works if configdir is under the git directory - which on test training is currently not true.

Comment (borg323) [reply tf/start.sh]: This is even better. Then we can do:
```
echo "Saving .yaml changes:"
pushd $CONFIGDIR
git add $netarch.yaml
git commit -m "Configuration change"
git push https://github.com/LeelaChessZero/lczero-training-conf.git &
popd
```
and have all changes in a new "lczero-training-conf" git repository

Comment (Error323) [reply tf/start.sh]: Sounds good to me!

**Pull request merged**